core/client/src/main/java/alluxio/client/file/options/CreateFileOptions.java

```diff
@@ -90,6 +93,14 @@ public long getTtl() {
   }
 
   /**
+   * @return the {@link TtlExpiryAction}; It informs the action to take when Ttl is expired. It can
```

Comment:
style: `the {@link TtlExpiryAction}`

(I would not enumerate the types of actions here as that information will become stale over time)